way_to_home/way_to_home/settings.py

```diff
@@ -155,6 +155,12 @@
 JWT_TOKEN_KEY = 'any secret word'
 JWT_ALGORITHM = 'HS384'
 
+# Required setting for NEXMO
+
+NEXMO_API_KEY = 'Nexmo API KEY'
```

Comment:
add these keys to local settings

Reply:
This has already been added. The localsettings will overwrite these values.